geopandas/tests/test_sindex.py

```diff
@@ -45,9 +45,7 @@ def test_no_sindex_active(self):
 class TestSeriesSindex:
     def test_empty_geoseries(self):
         """Tests creating a spatial index from an empty GeoSeries."""
-        with pytest.warns(FutureWarning, match="Generated spatial index is empty"):
-            # TODO: add checking len(GeoSeries().sindex) == 0 once deprecated
```

Comment:
Is this checked somewhere? Since we were recommending it in the warning, we should make sure it works.

Reply:
The warning is getting deleted by this PR. If you are referring to the TODO, good catch, I should have implemented that test. Added now.

Reply:
Yes, I meant TODO. And I think you wanted to push the commit :).

Reply:
Replied below